controllers/snapshot/snapshot_adapter.go

```diff
@@ -554,13 +542,25 @@ func (a *Adapter) createIntegrationPipelineRun(application *applicationapiv1alph
 
 	err := ctrl.SetControllerReference(snapshot, pipelineRun, a.client.Scheme())
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to set snapshot %s as ControllerReference of pipelineRun: %w", snapshot.Name, err)
 	}
 	err = a.client.Create(a.context, pipelineRun)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to call client.Create to create pipelineRun for snapshot %s: %w", snapshot.Name, err)
 	}
 
+	a.logger.LogAuditEvent("IntegrationTestscenario pipeline has been created", pipelineRun, h.LogActionAdd,
+		"integrationTestScenario.Name", integrationTestScenario.Name)
+	gitops.PrepareToRegisterIntegrationPipelineRun(a.snapshot)
+	if gitops.IsSnapshotNotStarted(a.snapshot) {
+		_, err := gitops.MarkSnapshotIntegrationStatusAsInProgress(a.client, a.context, a.snapshot, "Snapshot starts being tested by the integrationPipelineRun")
+		if err != nil {
+			a.logger.Error(err, "Failed to update integration status condition to in progress for snapshot")
```

Comment:
Can we include snapshot iitself here? So its easier to search for it.

Reply:
I think that the Snapshot that's being reconciled will always be reported in the logs on account of the `r.Log.WithValues` call [here](https://github.com/redhat-appstudio/integration-service/blob/d302862916d13621967f34c14f5d91f46dac8d28/controllers/snapshot/snapshot_controller.go#L78).

Reply:
Thanks Kruno for reply. Yes.